source/fuzz/transformation_replace_parameter_with_global.cpp

```diff
@@ -120,6 +116,14 @@ void TransformationReplaceParameterWithGlobal::Apply(
       fuzzerutil::MaybeGetZeroConstant(ir_context, *transformation_context,
                                        param_inst->type_id()));
 
+  // Mark the global variable's pointee as irrelevant if replaced parameter is
+  // irrelevant.
+  if (transformation_context->GetFactManager()->IdIsIrrelevant(
+          message_.parameter_id())) {
+    transformation_context->GetFactManager()->AddFactValueOfPointeeIsIrrelevant(
+        message_.global_variable_fresh_id());
+  }
+
```

Comment:
Do you have a test in a forthcoming PR that will showcase this functionality working?

Reply:
Yes, we've decided to make a separate PR to adjust tests.